sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -736,6 +736,9 @@ public static ClassificationCategoryCollection ClassificationCategoryCollection(
         }
         #endregion
 
+        #region Dynamic Classify
+        #endregion
```

Comment:
Just double checking, but you intend to add factory methods in a future PR. Is that right?

Reply:
Oh, funny. I suspect I lost this change because I might have forgotten to save the file before I rebased my branch. 🤔 I have added it back. Thank you for catching it!